app/src/main/java/ca/marshallasch/veil/database/Migrations.java

```diff
@@ -209,6 +209,75 @@ static void upgradeV9(SQLiteDatabase db) {
      * @param db the database object
      */
     static void upgradeV10(SQLiteDatabase db){
+
+        class KnownPostObject{
+            String postHash;
+            String commentHash;
+            int readStatus;
+            Long insertTime;
+
+            KnownPostObject(String _postHash, String _commentHash, int _readStatus, Long _insertTime){
```

Comment:
Why are the parameters named with underscores? It does not match the conventions in the rest of the code.

Reply:
rename to paramPostHash etc.